packages/react-noop-renderer/src/ReactNoop.js

```diff
@@ -47,9 +47,9 @@ export const {
   act,
   dumpTree,
   getRoot,
-  // TODO: Remove this once callers migrate to alternatives.
-  // This should only be used by React internals.
+  // TODO: Remove these after callers migrate to alternatives.
   unstable_runWithPriority,
+  unstable_InputContinuousLanePriority,
```

Comment:
Same comment as before. These are only used internally. Hardcode the number wherever it’s used, with a comment and a name that’s easily searchable.